app/questionnaire/router.py

```diff
@@ -350,7 +350,15 @@ def _get_return_to_for_calculated_summary(
             # remove first item and return the remaining ones
             # Type ignore: return_location.return_to and return_location.return_to_answer_id will always be populated at this point
             return_to = ",".join(return_location.return_to.split(",")[1:]) or None  # type: ignore
```

Comment:
Is this type ignore valid and guarded? We need to be sure otherwise this can also error.

Reply:
Yeah, it is guaranteed, this code only executes when higher up this `if` statement is valid:
```python
if return_location.return_to.startswith("calculated-summary")..
```